### PR TITLE
Make Tailwind & Turbolinks integration descriptions more consistent

### PR DESCRIPTION
**Pull request by Yan-Thomas**
## Changes

This PR changes the `package.json` description for the Tailwind and Turbolinks integrations to make them more consistent with the Partytown one.

## Testing

No code changes!

## Docs

It's all docs!
/cc @withastro/maintainers-docs

### CODE REVIEW

**Review by sarah11918 — APPROVED** [review]
A nit after my own heart.  💜

**Comment by Jutanium** [comment]
It's beautiful 🥹

File: packages/integrations/tailwind/package.json
Comment (delucis) [inline packages/integrations/tailwind/package.json:3]:
```suggestion
  "description": "Use Tailwind CSS to style your Astro site",
```

File: packages/integrations/turbolinks/package.json
Comment (delucis) [inline packages/integrations/turbolinks/package.json:3]:
```suggestion
  "description": "Deprecated — Use Turbolinks to speed up page navigation in your Astro site",
```

**Review by delucis — COMMENTED** [review]
I love it! Thank you @Yan-Thomas 💜 

We should add a patch changeset to this PR so that these updates also make it to npm.

One thing we could think about looking at https://astro.build/integrations/official/ is making these a bit more descriptive too.

Some examples of other descriptions there:

> Use React components within Astro

> Faster page navigations by prefetching links when the browser is idle.

I’ve made some suggestions and also suggested adding “Deprecated” to the Turbolinks description. Happy for you to make the final call though!

File: packages/integrations/turbolinks/package.json
Comment (Yan-Thomas) [reply packages/integrations/turbolinks/package.json:3]: Maybe we should even hide it from the Integrations catalogue? But sure, as a followup PR

**Comment by Yan-Thomas** [comment]
Marking this as a draft so we can discuss more about these descriptions internally before merging.

**Comment by Yan-Thomas** [comment]
This should be good now 👍, will add another PR updating other integrations titles.

**Review by natemoo-re — APPROVED** [review]
Thanks Yan!

**Pull request merged**
